src/main.py

```diff
@@ -263,6 +263,9 @@ def __init__(self, aconfig):
 
 		self._aconfig = aconfig
 
+		if not self._aconfig['refresh_token']:
```

Comment:
If `_aconfig` does not have that key, how does this prevents `Token` to progress? I think it is better to log warning, add that key and go on.